addOns/graaljs/src/main/zapHomeFiles/scripts/templates/active/Active default template GraalJS.js

```diff
@@ -22,6 +22,11 @@ alertTags:
   name2: value2
 otherInfo: Any other Info
 status: alpha
+alertRefOverrides:
+  12345-1: {}
```

Comment:
Does this functionally mean that -1 uses the ummm "base" details?

Reply:
Yes